subscriptions/server/internal/handler/subscriptions.go

```diff
@@ -0,0 +1,81 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"main/models"
+	"net/http"
+)
+
+func Subscriptions(m models.Fetcher) http.HandlerFunc {
```

Comment:
this approach is possible but has a disadvantage that transport layer and business logic, such as data validation and so on, are mixed. 
for example, if project migrate to another transport it would require to reimplement data validation as well.